src/Android/Xamarin.Android/Samples/Geometry/FormatCoordinates/FormatCoordinates.cs

```diff
@@ -0,0 +1,201 @@
+            _myMapView.GeoViewTapped += (sender, args) => { UpdateUiFromMapPoint(args.Location); };
+        }
+
+        private void InputTextChanged(object sender, EventArgs e)
+        {
+            // Keep track of the last edited field
+            _lastEdited = (EditText)sender;
+        }
+
+        private void ProcessTextChange(object sender, EventArgs e)
+        {
+            // Hold the entered point
+            MapPoint enteredPoint = null;
+
+            // Update the point based on which textbox sent the event
+            try
+            {
+                switch (_lastEdited.Hint)
+                {
+                    case "Decimal Degrees":
+                    case "Degrees, Minutes, Seconds":
+                        enteredPoint =
+                            CoordinateFormatter.FromLatitudeLongitude(_lastEdited.Text, _myMapView.SpatialReference);
+                        break;
+
+                    case "UTM":
+                        enteredPoint =
+                            CoordinateFormatter.FromUtm(_lastEdited.Text, _myMapView.SpatialReference, UtmConversionMode.LatitudeBandIndicators);
+                        break;
+
+                    case "USNG":
+                        enteredPoint =
+                            CoordinateFormatter.FromUsng(_lastEdited.Text, _myMapView.SpatialReference);
+                        break;
+                }
+            }
+            catch (Exception)
+            {
+                // The coordinate is malformed, return
+                // Sample doesn't handle this because coordinates can be invalid while the user is experimenting
+                return;
+            }
+
+            // Return if getting the point from text failed
+            if (enteredPoint == null)
+            {
+                return;
+            }
+
+            // Update the UI from the MapPoint
+            UpdateUiFromMapPoint(enteredPoint);
+        }
+
+        private void UpdateUiFromMapPoint(MapPoint startingPoint)
+        {
+            // Clear event subscriptions - prevents an infinite loop
+            _UtmEditText.TextChanged -= InputTextChanged;
+            _DmsEditText.TextChanged -= InputTextChanged;
+            _DecimalDegreesEditText.TextChanged -= InputTextChanged;
+            _UsngEditText.TextChanged -= InputTextChanged;
+
+            // Update the decimal degrees textbox
+            _DecimalDegreesEditText.Text =
+                CoordinateFormatter.ToLatitudeLongitude(startingPoint, LatitudeLongitudeFormat.DecimalDegrees, 4);
+
+            // Update the degrees, minutes, seconds textbox
+            _DmsEditText.Text = CoordinateFormatter.ToLatitudeLongitude(startingPoint,
+                LatitudeLongitudeFormat.DegreesMinutesSeconds, 1);
+
+            // Update the UTM textbox
+            _UtmEditText.Text = CoordinateFormatter.ToUtm(startingPoint, UtmConversionMode.LatitudeBandIndicators, true);
```

Comment:
I would suggest using `UtmConversionMode.NorthSouthIndicators` instead of `UtmConversionMode.LatitudeBandIndicators`. I think the N/S zone indicator is the more familiar way (for most folks) of viewing these coordinates.